stations.csv

```diff
@@ -3426,7 +3426,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 4571;Joué-lès-Tours;joue-les-tours;8757188;87571885;47.3541741;0.6675586;4580;FR;Europe/Paris;f;t;f;t;f;f;FRJLT;JLT;t;;f;8703509;f;;f;;f;;f;;f;;f;8703509;f;;f;;f;;;;f;;f;;f;;f;;f;;f;t;;;;;;;;;;;;;;;;Жу-ле-Тур;;;;urn:trainline:public:nloc:csv4571;
 4572;Job;job;8773169;87731695;45.6166670000;3.7500000000;;FR;Europe/Paris;f;f;f;t;f;f;FRJOB;;t;;f;8703502;f;;f;;f;;f;;f;;f;8703502;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4572;
 4573;Josselin;josselin;8732200;87322008;47.9500000000;-2.5500000000;;FR;Europe/Paris;f;f;f;t;f;f;FRJOS;;t;;f;8701207;f;;f;;f;;f;;f;;f;8701207;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;ジョスラン;;;;Жослен;;;若瑟兰;urn:trainline:public:nloc:csv4573;
-4575;Juvisy TGV;juvisy-tgv;8749669;87496695;48.689496;2.383498;10715;FR;Europe/Paris;f;f;f;f;f;f;FRJSH;JYG;f;;f;8706453;f;;f;;f;;f;;f;;f;8700352;f;;f;;f;;;;f;;f;;f;;f;;f;;f;t;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4575;
+4575;Paris Gare du Nord Surface;paris-gare-du-nord;8727100;87271007;48.880185;2.355151;4916;FR;Europe/Paris;f;f;f;f;f;f;FRPNB;PNB;f;;f;8700014;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;FRCCO;f;;f;f;4922;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4922;
```

Comment:
:thinking: We should not be reusing the same `id` for a completely different station. Can you rather create a new one?